src/govuk/components/exit-this-page/exit-this-page.mjs

```diff
@@ -1,21 +1,50 @@
 /* eslint-disable es-x/no-function-prototype-bind -- Polyfill imported */
 
-import { nodeListForEach } from '../../common.mjs'
+import { normaliseDataset } from '../../common/normalise-dataset.mjs'
+import { nodeListForEach, mergeConfigs, extractConfigByNamespace } from '../../common.mjs'
+import { I18n } from '../../i18n.mjs'
 import '../../vendor/polyfills/Element/prototype/classList.mjs'
 import '../../vendor/polyfills/Element/prototype/dataset.mjs'
 import '../../vendor/polyfills/Function/prototype/bind.mjs'
 
+/**
+ * @constant
+ * @type {ExitThisPageTranslations}
+ * @see Default value for {@link ExitThisPageConfig.i18n}
+ * @default
+ */
+var EXIT_THIS_PAGE_TRANSLATIONS = {
```

Comment:
A general comment that even if we end up putting this work down for now, we should retain this as it's good for getting the final code up to scratch.